tests/e2e/examples/source/dotnet/Startup.cs

```diff
@@ -29,7 +29,7 @@ public static void Main(string[] args)
                     options.NoDelay = true;
                     options.UseConnectionLogging();
                 })
-                .UseUrls("http://0.0.0.0:8080")
+                .UseURLs("http://0.0.0.0:8080")
```

Comment:
This is not correct, it breaks the source e2e test :)

Reply:
ack